README.md

```diff
@@ -24,7 +24,7 @@ Let's say you have the following `SampleEndpoints` implementation:
 ```typescript
 import { RequestHandler } from 'express'
 
-const helloWorld: RequestHandler = (req, res, next) => {
+const helloWorld: RequestHandler<any, any, any, { name: string } = (req, res, next) => {
```

Comment:
We cannot solve the generics issue by just changing the RequestHandler types passed to the `test` / `testError` methods. - this will not happen in real usage scenarios. Instead we need to allow any type of RequestHandler / ErrorRequestHandler to be passed.